src/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/CommandID.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+
+
+
+/*
+ */
+
+namespace System.ComponentModel.Design
+{
+    using System;
+    using System.ComponentModel;
+    using System.Diagnostics;
+    using System.Globalization;
+    using System.Security.Permissions;
+
+    /// <summary>
+    ///    <para>
+    ///       Represents a
+    ///       numeric Command ID and globally unique
+    ///       ID (GUID) menu identifier that together uniquely identify a command.
+    ///    </para>
+    /// </summary>
+    [HostProtection(SharedState = true)]
+    
+    [System.Security.Permissions.PermissionSetAttribute(System.Security.Permissions.SecurityAction.InheritanceDemand, Name = "FullTrust")]
+    [System.Security.Permissions.PermissionSetAttribute(System.Security.Permissions.SecurityAction.LinkDemand, Name = "FullTrust")]
+    public class CommandID
+    {
+        private readonly Guid _menuGroup;
+        private readonly int _commandID;
+
+        /// <summary>
+        ///    <para>
+        ///       Initializes a new instance of the <see cref='System.ComponentModel.Design.CommandID'/>
+        ///       class. Creates a new command
+        ///       ID.
+        ///    </para>
+        /// </summary>
+        public CommandID(Guid menuGroup, int commandID)
+        {
+            _menuGroup = menuGroup;
+            _commandID = commandID;
+        }
+
+        /// <summary>
+        ///    <para>
+        ///       Gets or sets the numeric command ID.
+        ///    </para>
+        /// </summary>
+        public virtual int ID
+        {
+            get
+            {
+                return _commandID;
+            }
+        }
```

Comment:
Nit: another example where a C#6 feature helps to simplify... instead of having both ID and _commandID, it can just be:
```C#
public virtual int ID { get; }
```